FileConv/Gfx/PrintShop-notes.md

```diff
@@ -39,13 +42,42 @@ the cyan plane.
 The actual color palette, as determined by a screen grab from an emulator, is comprised of RGB
 primaries and purple/orange that are similar to the classic Apple II hi-res colors.
 
-Y M C | color  | red, green, blue
------ | ------ | ----------------
-0 0 0 | white  | 0xff, 0xff, 0xff
-0 0 1 | blue   | 0x00, 0x00, 0xff
-0 1 0 | red    | 0xff, 0x00, 0x00
-0 1 1 | purple | 0xcc, 0x00, 0xcc
-1 0 0 | yellow | 0xff, 0xff, 0x00
-1 0 1 | green  | 0x00, 0xff, 0x00
-1 1 0 | orange | 0xff, 0x66, 0x00
-1 1 1 | black  | 0x00, 0x00, 0x00
+| Y M C | color  | red, green, blue |
+| ----- | ------ | ---------------- |
+| 0 0 0 | white  | 0xff, 0xff, 0xff |
+| 0 0 1 | blue   | 0x00, 0x00, 0xff |
+| 0 1 0 | red    | 0xff, 0x00, 0x00 |
+| 0 1 1 | purple | 0xcc, 0x00, 0xcc |
+| 1 0 0 | yellow | 0xff, 0xff, 0x00 |
+| 1 0 1 | green  | 0x00, 0xff, 0x00 |
+| 1 1 0 | orange | 0xff, 0x66, 0x00 |
+| 1 1 1 | black  | 0x00, 0x00, 0x00 |
+
+## Fonts
+
+Each font contains 59 characters (0-58).
+
+In the Print SHop Companion (PSC) font editor, offsets 0 (space) and 32 (image) are not editable.
+
+In Print Shop[, offset 32 points to the current graphic selected, which
+is generally not contained in the font file itself.  However, since it is an arbitrary data pointer, it is
+conceivble that there could be self-contained data in the file.
+
+The Space character is at offset 0, and in all the examples I've seen, has a height of 0 pixels.
+The data is calculated in the PSC font editor based off of an average of all other character widths.
+
+$5FF4-$5FFF - Optional Print Shop Companion Font Header (From Font Editor)
```

Comment:
The way I've done this stuff in the various -notes.md files is "+offset / length" rather than memory address, since that's what you'll be looking at if you dump the contents of the file as hex.  I initially did start/end offset for things like ProDOS directory structures, but found myself constantly doing mental math to figure out how many bytes wide a field was.

So this might be written:
+$00 / 10: optional Print Shop ...
+$0a / 59: character widths (in pixels)...
+$45 / 59: character heights...
+$80 / 59: low bytes...
+$bb / 59: high bytes...
+$f6 / nn: character data

This way, the various fields that are 59 bytes jump out at you, making it obvious that it's multiple parallel per-character entries.

If the PSC font header is a fixed string, you can document it here.  If it's uncommon you can skip it, and just document the format starting at +$00 for character widths, and just note that there may be a 10-byte header before it all.